client/settings/payment-request-section/index.js

```diff
@@ -1,178 +1,205 @@
-								disabled={ ! isPaymentRequestEnabled }
-								checked={
-									isPaymentRequestEnabled &&
-									paymentRequestLocations.includes( 'cart' )
+							</div>
+							<div className="express-checkout__description">
+								{
+									/* eslint-disable jsx-a11y/anchor-has-content */
+									interpolateComponents( {
+										mixedString: __(
+											'Boost sales by offering a fast, simple, and secure checkout experience.' +
+												'By enabling this feature, you agree to {{stripeLink}}Stripe{{/stripeLink}}, ' +
+												"{{appleLink}}Apple{{/appleLink}}, and {{googleLink}}Google{{/googleLink}}'s terms of use.",
+											'woocommerce-gateway-stripe'
+										),
+										components: {
+											stripeLink: (
+												<a
+													target="_blank"
+													rel="noreferrer"
+													href="https://stripe.com/apple-pay/legal"
+												/>
+											),
+											appleLink: (
+												<a
+													target="_blank"
+													rel="noreferrer"
+													href="https://developer.apple.com/apple-pay/acceptable-use-guidelines-for-websites/"
+												/>
+											),
+											googleLink: (
+												<a
+													target="_blank"
+													rel="noreferrer"
+													href="https://androidpay.developers.google.com/terms/sellertos"
+												/>
+											),
+										},
+									} )
+									/* eslint-enable jsx-a11y/anchor-has-content */
 								}
-								onChange={ makeLocationChangeHandler( 'cart' ) }
-								label={ __(
-									'Cart',
+							</div>
+						</div>
+						<div className="express-checkout__link">
+							<a href={ customizeAppearanceURL }>
+								{ __(
+									'Customize',
 									'woocommerce-gateway-stripe'
 								) }
-							/>
+							</a>
+						</div>
+					</li>
+					{ displayLinkPaymentMethod && (
+						<li className="express-checkout has-icon-border">
+							<div className="express-checkout__checkbox loadable-checkbox label-hidden">
+								<CheckboxControl
+									label={ __(
+										'Link by Stripe Input',
+										'woocommerce-payments'
+									) }
+									checked={ isStripeLinkEnabled }
+									onChange={ updateStripeLinkCheckout }
+								/>
+							</div>
+							<div className="express-checkout__icon">
+								<LinkIcon size="medium" />
+							</div>
+							<div className="express-checkout__label-container">
+								<div className="express-checkout__label">
+									{ __(
+										'Link by Stripe',
+										'woocommerce-gateway-stripe'
+									) }
+								</div>
+								<div className="express-checkout__description">
+									{
+										/* eslint-disable jsx-a11y/anchor-has-content */
+										interpolateComponents( {
+											mixedString: __(
+												'Link autofills your customers’ payment and shipping details to ' +
+													'deliver an easy and seamless checkout experience. ' +
+													'New payment experience (UPE) needs to be enabled for Link. ' +
```

Comment:
I think this is called ["New checkout experience"](https://woocommerce.com/document/stripe/#new-checkout-experience) in Stripe, and users might not be familiar with the term UPE 🤔.